### PR TITLE
Updated to be compliant with Cyber Essentials Plus

### PR DESCRIPTION
**Pull request by Rucknar**

### CODE REVIEW

File: ISO-27001/computing/readme.md [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
+- This policy applies to all Infinity Works employees, contractors, vendors and agents with an Infinity Works-owned device.
+- This policy applies to all contractors working on behalf of Infinity Works using a device provided by themselves for the role, only when using the device to access Infinity Works systems.
+
+This policy covers all computers, servers, smartphones, tablets and other computing devices operating within Infinity Works.
 
 ## Policy
 
-- Employees may install software on computing devices operated within the Infinity Works network in order to perform their role / function.
+- Employees will have the ability to perform administrative actions on their device, unless explicitly revoked on an individual level by the ISMS.
+- The main user logins on a machine should not be running with super user priveledges, users should maintain a seperate administrative account for when that level of access is needed.
```
Comment (timslow) [inline ISO-27001/computing/readme.md:21]: For compliance we should also state that the Administrator account must not to be used for accessing the internet or email.

File: ISO-27001/networksystems/readme.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ This policy applies to all networks within Infinity Works offices.
...
 - The ISMS Committee has the authority to remove from the network any equipment which is interfering with the network service or is deemed likely to compromise the security of the network.
-- Anyone connecting equipment to the network is responsible for ensuring that the equipment is configured correctly, that the operating systems and software applications are up-to-date as regards patch management etc. and that the equipment has adequate protection against viruses and other malware. If there is any suspicion that the equipment may be infected or compromised in any way it should not be connected.
+- Anyone connecting equipment to the network is responsible for ensuring that the equipment is configured correctly, that the operating systems and software applications are patched with the latest available security update (See [computing policy](../computing/readme.md). And that the equipment has adequate protection against viruses and other malware through use of company issued ESET malware control. If there is any suspicion that the equipment may be infected or compromised in any way it should not be connected.
```
Comment (timslow) [inline ISO-27001/networksystems/readme.md:20]: Could we add in a comment about Web control protection has to be enabled for ESET.
Also we need staff to ensure local firewall is enabled.
Can we perhaps add in device Encryption, password on the device and Google Endpoint verification is run on devices and checked every 30 days?

Comment (Rucknar) [reply ISO-27001/networksystems/readme.md:20]: Sure, could you give me some wording? I'll push a commit.

File: ISO-27001/computing/readme.md (hunk 1 above)
Comment (Rucknar) [reply ISO-27001/computing/readme.md:21]: Good shout

File: ISO-27001/computing/readme.md [hunk 3]
```diff
@@ -1,37 +1,46 @@
...
+- Employees may install software on computing devices operated within the Infinity Works network in order to perform their role / function, using the seperated admin account.
 - All software must be appropriately licensed either by Infinity Works or by the user.
+- Operating system and application security patches must be applied within 14 days maximum of being made available.
+- All laptops and desktops must run company issued malware protection, above and beyond the built-in OS controls (ESET).
```
Comment (timslow) [inline ISO-27001/computing/readme.md:25]: Suggest revise this to ...
All laptops and desktops must run company issued malware protection (ESET) with daily updates enabled and the web and email protections enabled, this is above and beyond the built-in OS controls.
All laptops and desktops must have the local OS firewall enabled.
All laptops and desktops must have full disk Encryption enabled, Filevault on Mac and Bitlocker on Windows.
All laptops and desktops must have the Google Endpoint Verification plugin enabled and must run a sync at least every 30 days.

File: ISO-27001/networksystems/readme.md (hunk 2 above)
Comment (timslow) [reply ISO-27001/networksystems/readme.md:20]: Added further comment to the Computing policy with these suggested changes, I think this network policy is actually fine and just changes for the Computing policy.

**Review by timslow — COMMENTED** [review]
This looks good to me.

File: ISO-27001/computing/readme.md [hunk 4]
```diff
@@ -18,15 +18,19 @@ This policy covers all computers, servers, smartphones, tablets and other comput
 ## Policy
 
 - Employees will have the ability to perform administrative actions on their device, unless explicitly revoked on an individual level by the ISMS.
-- The main user logins on a machine should not be running with super user priveledges, users should maintain a seperate administrative account for when that level of access is needed.
+- The main named user login on a machine should not be running with super user rights.
+- Users should maintain a seperate administrative account for when that level of access is needed, this account can either be used to authorise priveledged operations when prompted, or switched into to perform an action. This user must not to be used for accessing the internet or email.
```
Comment (timslow) [inline ISO-27001/computing/readme.md:22]: Suggest revision to ... Users should maintain a separate administrative account for when that level of access is needed, this account can either be used to authorise privileged operations when prompted, or switched into to perform an action. This user must not to be used for accessing the internet or email. This admin account must be password enabled and use a different password to the standard user account.

File: ISO-27001/computing/readme.md [hunk 5]
```diff
@@ -18,15 +18,19 @@ This policy covers all computers, servers, smartphones, tablets and other comput
 ## Policy
 
 - Employees will have the ability to perform administrative actions on their device, unless explicitly revoked on an individual level by the ISMS.
-- The main user logins on a machine should not be running with super user priveledges, users should maintain a seperate administrative account for when that level of access is needed.
+- The main named user login on a machine should not be running with super user rights.
```
Comment (timslow) [inline ISO-27001/computing/readme.md:21]: The main named user login on a machine should not be running with super user rights. This standard user account must be password protected with a password length of at least 8 characters.

**Review by timslow — COMMENTED** [review]
Added suggested revisions around password difference between the 2 local accounts and password being at least 8 characters for both accounts.

File: ISO-27001/computing/readme.md (hunk 5 above)
Comment (hindle) [reply ISO-27001/computing/readme.md:21]: Slight tweak to that.

The main named user login on a machine should not be running with super user rights; for Mac / Linux machines, this user should not be added to the sudoers file. This standard user account must be password protected using a password that adheres to the [password policy](/ISO-27001/password/).

Suggest we refer to the password policy so this doesn't need to keep being updated. I'll raise a change to tweak the password policy to reflect the minimum 8 characters.

File: ISO-27001/computing/readme.md (hunk 4 above)
Comment (timslow) [reply ISO-27001/computing/readme.md:22]: A few other requirements around turning off Auto play, believe this only affects Windows, suggest also adding the following, also some other suggested additions to the policy from Cyber Essentials further review ...
"Autoplay"/"Auto-run" options must be disabled on the laptop/desktop.
Any default user account for any device, must be disabled or password changed on the device.
Any Application that is no longer supported and no longer has security fixes updates applied must be removed from the device.
All critical security firmware updates must be applied on the device within 14 days.

File: ISO-27001/computing/readme.md [hunk 6]
```diff
@@ -1,37 +1,50 @@
...
 
 ## Policy
 
-- Employees may install software on computing devices operated within the Infinity Works network in order to perform their role / function.
+- Employees will have the ability to perform administrative actions on their device, unless explicitly revoked on an individual level by the ISMS.
+- The main named user login on a machine should not be running with super user rights.
+- Users should maintain a seperate administrative account for when that level of access is needed, this account can either be used to authorise priveledged operations when prompted, or switched into to perform an action. This user must not to be used for accessing the internet or email.
+- Employees may install software on computing devices operated within the Infinity Works network in order to perform their role / function, using the seperated admin account.
 - All software must be appropriately licensed either by Infinity Works or by the user.
+- Operating system and application security patches must be applied within 14 days maximum of being made available.
```
Comment (hindle) [inline ISO-27001/computing/readme.md:25]: Operating system, firmware and and application security patches must be applied within 14 days of being made available.

File: ISO-27001/computing/readme.md [hunk 7]
```diff
@@ -1,37 +1,50 @@
...
+- All laptops and desktops must run company issued malware protection (ESET) with daily updates enabled and the web and email protections enabled, this is above and beyond the built-in OS controls.
+- All laptops and desktops must have the local OS firewall enabled.
+- All laptops and desktops must have full disk Encryption enabled, Filevault on Mac and Bitlocker on Windows.
 - All users must take every measure to ensure that they do not inject malware onto Infinity Works systems and networks.
 - The Administration Team will obtain and track licenses owned by Infinity Works.
+- All non-essential software should be removed when no longer relevant or required to ensure the security footprint of the device is minimised.
```
Comment (hindle) [inline ISO-27001/computing/readme.md:31]: Additional sentence:

Any application that is no longer supported or no longer receive security patches should be removed.

File: ISO-27001/computing/readme.md (hunk 4 above)
Comment (Rucknar) [reply ISO-27001/computing/readme.md:22]: @timslow Given we know the state of the machines. Do we really have to then ask consultants to disable standard user accounts?  If there isn't any, we should adjust the wording, i don't want people feeling like day one of the role is reconfiguring windows etc.

Comment (Rucknar) [reply ISO-27001/computing/readme.md:22]: Same for auto-play, is it on by default?

Comment (Rucknar) [reply ISO-27001/computing/readme.md:22]: In-fact auto-play has been deprecated by MS so doesn't need to be added.

File: ISO-27001/computing/readme.md (hunk 5 above)
Comment (Rucknar) [reply ISO-27001/computing/readme.md:21]: Spot on Anthony

File: ISO-27001/computing/readme.md (hunk 4 above)
Comment (Rucknar) [reply ISO-27001/computing/readme.md:22]: Sorry, auto-run has been deprecated. Not auto-play

Comment (timslow) [reply ISO-27001/computing/readme.md:22]: As both Mac and Windows don't come with na account already on vanilla OS (that is part of the setup), then we're compliant on this, so we can remove that requirement.
Auto-play does need to be disabled, I'll install a fresh windows and see it's default state.

Comment (timslow) [reply ISO-27001/computing/readme.md:22]: Auto-play does appear to be on by default on Windows.

**Review by timslow — APPROVED** [review]

**Pull request merged**